tests/gdx-tests-android/src/com/badlogic/gdx/tests/android/AndroidTestStarter.java

```diff
@@ -38,6 +37,8 @@
 	public void onCreate (Bundle savedInstanceState) {
 		super.onCreate(savedInstanceState);
 		GdxTests.tests.add(MatrixTest.class);
+		if (!GdxTests.tests.contains(APKExpansionTest.class))

```

Comment:
Why exactly do you do this?
If you want to add a new test you should add it to list in com.badlogic.gdx.tests.utils.GdxTests.